### ISSUE TITLE
Deploy Gemfile.lock to Docker Containers

### ISSUE THREAD

**Issue by jason-klein** [shannonmcmurtrey/nwp#5]
New to Ruby? [Learn more about Gemfile and Gemfile.lock files here](https://bundler.io/v1.3/rationale.html)

TL;DR - The project maintainer should occasionally upgrade Gem versions, test the project to ensure the new versions do not break functionality, and then commit the updated `Gemfile` and `Gemfile.lock` files to Git source control. Both files should be deployed to servers to ensure that the servers are running known working versions of each module.

**Project Issue**

If you have previously built this project and you run `docker-compose up --build` again, you may encounter Gem module errors about missing files or modules. I can confirm that deleting the “Gemfile.lock” file and running `docker-compose up --build` again **does** resolve the issue.

This issue occurs when a new version of a Gem is released. For example, when I first built the project, bootstrap_form-4.0.0 was installed. When I built the project again, I started to receive an error that the bootstrap_form-4.1.0 module could not be found.

I realized bootstrap_form-4.1.0 had been released and Ruby is attempting to use this latest version of the module, even though the latest version has not been installed to the container. Copying the `Gemfile.lock` file into the Docker containers will resolve the problem by forcing the container to always install and use the module versions that are specified in the `Gemfile.lock` file.

**Attempted Solution**

I would have expected the following Dockerfile command would have copied the Gemfile.lock into the container:

`COPY Gemfile.lock ./`

Instead, I received the following error when I ran the `docker-compose up --build` command:

`Step 6/10 : COPY Gemfile.lock ./
ERROR: Service 'website' failed to build: COPY failed: stat Gemfile.lock: no such file or directory`

I am unsure why this approach is not working. Can someone please help figure out how to deploy the `Gemfile.lock` file into the containers?
